electricitymap/contrib/config/__init__.py

```diff
@@ -173,6 +44,27 @@ def _can_zones_be_neighbours(
         for sub_zone_id in zone_config["subZoneNames"]:
             ZONE_PARENT[sub_zone_id] = zone_id
 
+# This object represents the edges of the flow-tracing graph
+ZONE_NEIGHBOURS: Dict[ZoneKey, List[ZoneKey]] = {}
```

Comment:
Can you replicate the structure I laid out with a generation function? 

This allows to unit test it and document more easily what the expected zone_neighbours object we get depending on the provided config file.

Reply:
Done. I did not unit test it though.

Reply:
Done